dev/bots/test.dart

```diff
@@ -116,8 +130,37 @@ Future<void> main(List<String> args) async {
   print('$clock ${bold}Test successful.$reset');
 }
 
+/// Verify the Flutter Engine is the revision inin
+/// bin/cache/internal/engine.version.
+Future<void> _validateEngineHash() async {
+  final String luciBotId = Platform.environment['SWARMING_BOT_ID'] ?? '';
+  if (luciBotId.startsWith('luci-dart-')) {
+    // The Dart HHH bots intentionally modify the local artifact cache
+    // and then use this script to run Flutter's test suites.
+    // Because the artifacts have been changed, this particular test will return
+    // a false positive and should be skipped.
+    print('${yellow}Skipping Flutter Engine Version Validation for swarming '
+          'bot $luciBotId.');
+    return;
+  }
```

Comment:
This is the new part from last time (minor some slight refactoring).  Bot ID was taken from https://logs.chromium.org/logs/dart/buildbucket/cr-buildbucket.appspot.com/8882487817844098672/+/steps/flutter_test_add_to_app_tests/0/logs/execution_details/0